index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/1ca239d8ab.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Version Portfolio</title>
+</head>
+<body>
+  <div class="container">
+
+    <section class="container-section">
+    <header>
+      <nav class="nav-header">
+        <div class="logo">My Logo</div>
+        <span class="bars"><i class="fa-solid fa-bars"></i></span>
+      </nav>
+    </header>
+    
+      <div class="intro">
```

Comment:
- [ ] For best [semantic practices](https://www.freecodecamp.org/news/html-best-practices/#:~:text=It%20is%20best%20practice%20to,and%20elements%20instead.), it is better to use a `section` tag for this part of your page rather than the generic `div` tag. Kindly correct this 👍